SkillChat.Client/Views/SendMessageControl.xaml.cs

```diff
@@ -7,15 +7,37 @@
 namespace SkillChat.Client.Views
 {
 	public class SendMessageControl : UserControl
-	{
+    {
+        private TextBox tb_message;
```

Comment:
Название нарушает общепринятую конвенцию именования
```suggestion
        private TextBox messageTextBox;
```